oe_theme.theme

```diff
@@ -2518,3 +2518,78 @@ function oe_theme_page_attachments_alter(array &$attachments) {
 function _oe_theme_sanitize_url($url): mixed {
   return $url instanceof Url ? $url : UrlHelper::filterBadProtocol($url);
 }
+
+/**
+ * Implements hook_preprocess_HOOK().
+ */
+function oe_theme_preprocess_pattern_highlighted_list(array &$variables): void {
```

Comment:
It looks like most of this code can be moved into template.

Reply:
This would keep the template cleaner, no?

Reply:
Normally before we tried to move simple manipulation with data inside template. But it is discussable. We could leave like that.

Reply:
We need to override too much data of the list_item pattern in order to make sure the data is correct for this pattern. I'd keep it in the preprocess.